### DISCUSSION TITLE
cozystack on hetzner cloud

### DISCUSSION THREAD

**Discussion by cgruenwald** [cozystack/cozystack#109] [Q&A]
Hi Guys

I really like your work and have already deployed cozystack in 'paas-full' flavour according your get started guide.
Now i am trying to deploy it in Hetzner Cloud.
I already tryed following the Hetzner How-To and I considered the guide on the Talos Site (https://www.talos.dev/v1.7/talos-guides/install/cloud-platforms/hetzner/) but seemingly my knowledge about Kubernetes and Networking is not enought to get those guides together and thngs working.

Maybe someone can kindly give me a hint on what are the cruzial configuration settings and steps to set up a nice cozystack in the hetzner cloud.

thanks in advance, keep your good work up

KR

Chris

**Comment by themoriarti** [comment]
Hi, Chris.

Thank you for your interest in the stack.

Do you want to start cozystack at three virtual servers on Hetzner Cloud?

If so, we haven't tested it on the Hetzner Cloud yet, but it should work. To do this, you need to create one of the servers and go to server menu -> Rescue -> Enable rescue & power cycle.
After this enter in server and execute commands:
```
cd /tmp
wget https://github.com/aenix-io/cozystack/releases/download/v0.2.0/metal-amd64.raw.xz
xz -d -c /tmp/metal-amd64.raw.xz | dd of=/dev/nvme0n1 bs=4M oflag=sync
```

our instruction is not designed to be copy paste, you need to read it in detail and change it to suit your situation.

If there are any questions, indicate at which step https://cozystack.io/docs/how-tos/hetzner/ and https://cozystack.io/docs/get-started/#bootstrap-cluster

**Comment by kvaps** [comment]
Hey, I just wanted to note that Hetzner Cloud is not the same as Hetzner Dedicated Servers

Talos has the inscruction on how to isntall Talos Linux on Hetzner Cloud
- https://www.talos.dev/v1.7/talos-guides/install/cloud-platforms/hetzner/

We have instuction on how to install Cozystack on Dedicated server
- https://cozystack.io/docs/how-tos/hetzner/
